### PR TITLE
add test example for network  loss, reorder, duplicate and corrupt in TiDB

### PR DESCRIPTION
**Pull request by WangXiangUSTC**

### CODE REVIEW

File: network/loss.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Test Network Chaos on TiDB
+
+## Network Loss
+
+### Description
+
+Test the availability of TiDB cluster in network package loss scenarios.
```
Comment (fewdan) [inline network/loss.md]: The above are all "the TiDB", and this should also be changed to be consistent with the above?

**Review by fewdan — APPROVED** [review]
rest LGTM

File: network/loss.md (hunk 1 above)
Comment (WangXiangUSTC) [reply network/loss.md]: done, PTAL again
